velocity/src/main/java/me/leoko/advancedban/velocity/command/CommandReceiverVelocity.java

```diff
@@ -0,0 +1,41 @@
+package me.leoko.advancedban.velocity.command;
```

Comment:
In accordance with the layout of other AdvancedBan environments (Bukkit and Bungee), perhaps you could move this to the `listener` package

Reply:
I've refactored this but I do believe that this should be moved to its own package as it's not a Listener